guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>In <em>part-1</em> of this guide series, we started with an introduction to the <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#geoaccessor" target="_blank"><strong>Spatially enabled DataFrame</strong></a> (SeDF), the <code>spatial</code> and <code>geom</code> namespaces, and looked at a quick example of SeDF in action. In this second part of the series, we will look at how GIS data can be accessed from various data formats using SeDF. </p><p>GIS users work with different vector-based spatial data formats, like published layers on remote servers (web layers) and local data. The <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#geoaccessor" target="_blank"><strong>Spatially enabled DataFrame</strong></a> allows the users to read, write, and manipulate spatial data by bringing the data in-memory. </p><p>The <em>SeDF</em> integrates with Esri's <a href="http://pro.arcgis.com/en/pro-app/arcpy/get-started/what-is-arcpy-.htm" target="_blank"><strong>ArcPy</strong> site-package</a>, as well as the open source <a href="https://github.com/GeospatialPython/pyshp/" target="_blank">pyshp</a>, <a href="https://github.com/Toblerity/Shapely" target="_blank">shapely</a> and <a href="https://github.com/Toblerity/Fiona" target="_blank">fiona</a> packages. This means that the <em>SeDF</em> can use either <a href="https://pypi.org/project/Shapely/" target="_blank">shapely</a> or <a href="https://www.esri.com/en-us/arcgis/products/arcgis-python-libraries/libraries/arcpy" target="_blank">arcpy</a> geometry engines to provide you with options for easily working with geospatial data, regardless of your platform.  The <em>SeDF</em> transforms the data into the formats you desire, allowing you to use Python functionality to analyze and visualize geographic information. </p><p>Data can be read and scripted to automate workflows and be visualized on maps in a <a href="https://github.com/Esri/arcgis-python-api/raw/4a67f22df96435cd793801397f6ab0034ae6f704/guide/using-the-jupyter-notebook-environment/" target="_blank">Jupyter notebooks</a>. Let's explore the options available for accessing GIS data with the versatile <em>Spatially enabled DataFrame</em>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>